man/shutdown.xml

```diff
@@ -80,7 +80,9 @@
         <term><option>-P</option></term>
         <term><option>--poweroff</option></term>
 
-        <listitem><para>Power the machine off (the default).</para></listitem>
+        <listitem><para>Power the machine off (the default). Note that the default in other systems 
+          (including sysvinit) was to fallback to single-user mode instead. Use 
+          <command>systemctl rescue</command> instead.</para></listitem>
```

Comment:
I like the idea of that compat section. Stuff that could be added there is 
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=884994#42

Unless it's planned to change the scheduled shutdown behaviour.

Reply:
> "fallback" is a noun. "fall back" is the verb-phrase that should be used instead.
> Also "Use … instead" is ambiguous: it sounds like systemctl rescue should be always used instead of -P.

I have rephrased and moved to the compatibility section, can you check again?

Reply:
> I like the idea of that compat section. Stuff that could be added there is https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=884994#42
> 
> Unless it's planned to change the scheduled shutdown behaviour.

hhmmm... that would be in a different PR though, right?

Reply:
Yes, no need to do that here, it can be added separately